src/pages/index.js

```diff
@@ -8,100 +8,62 @@ import styles from "./styles.module.css";
 
 const features = [
   {
-    title: "Guides",
-    imageUrl: "img/guides.png",
+    title: "I'm new here, and want to get started",
+    imageUrl: "img/confused-person.png",
     url: "/docs/guides",
-    description: (
-      <>
-        Step-by-step content to strengthen your knowledge of Camunda 8, its
-        components, and its features.
-      </>
-    ),
+    description:
+      "Visit our guides section to learn more about creating an account, modeling your first process, orchestrating human and service tasks, and more!",
   },
   {
-    title: "Components",
-    imageUrl: "img/components.png",
-    url: "/docs/components/",
-    description: (
-      <>
-        Conceptualize the cornerstones of Camunda 8 and learn more about its
-        supporting components.
-      </>
-    ),
+    title: "I'm looking for a particular component",
+    imageUrl: "img/homegrown-workflow.png",
+    url: "/docs/components/components-overview.md",
```

Comment:
Like the URL or linked page above it, this page has a slug in the metadata, so it will just be `/docs/components/` to match the slug value.